src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* BUGFIX: Fix merge bug in Linux when only pass file name without path. [#2408](https://github.com/microsoft/sarif-sdk/pull/2408)
```

Comment:
>Fix [](http://example.com/codeflow?start=10&length=3)

Add details here with a special focus on what symptoms a customer would observe if they hit this problem. Imagine what someone would query the internet for.

Reply:
added what issue user will see before fix.